.github/workflows/genai_python_lib.yml

```diff
@@ -33,6 +33,7 @@ jobs:
       - run: source ./ov/setupvars.sh && PYTHONPATH=./build/:$PYTHONPATH python -m pytest ./tests/python_tests/test_generate_api.py -m precommit
       - run: source ./ov/setupvars.sh && python -m pip install . --verbose
       - run: python -m pytest ./tests/python_tests/test_generate_api.py -m precommit
+      - run: python -m pytest ./tests/python_tests/test_chat_generate_api.py -m precommit
```

Comment:
done

Reply:
This covers continuous batching workflows added by @as-suvorov, please delete dedicated workflows testing python continuous batching part.

Why do you keep `-m precommit`? There's already `addopts = -m precommit` added by you in `pytest.ini`.

Reply:
Agreed, since `python -m pytest ./tests/python_tests/` covers them as well. Deleted separate jobs for continuous batching.

I used `-m precommit' by inertia :) Removed since it's not necessary